### PR TITLE
Basic file upload functionality with no restrictions on file type

### PR DESCRIPTION
**Pull request by jonathan-mejia-deluna**

### CODE REVIEW

**Comment by orogenic** [comment]
@jonathan-mejia-deluna Can you try doing two things 1) assign yourself to issue #1 2) link issue #1 to this PR (or vice-versa, idk, not sure how github works)

**Comment by jonathan-mejia-deluna** [comment]
> @jonathan-mejia-deluna Can you try doing two things 1) assign yourself to issue #1 2) link issue #1 to this PR (or vice-versa, idk, not sure how github works)

I think I've done it

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
 /venv/
+.env
+__pycache__/
```
Comment (orogenic) [inline .gitignore:3]: tempting to add everything in your worktree to the index and commit, but these changes aren't actually related to the issue. that said, it's early, no sweat. but let's practice the habit of thinking in terms of well-separated changes.

**Review by orogenic — APPROVED** [review]
convention for flask is to put everything in a `/static` directory. I don't see the functional reason to separate files into subdirectories by type, so based on "don't do anything 'til it's necessary", i'd avoid the additional structure. just minor preferences. Approved.

File: html/uploadfile.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Upload File</title>
+    <script src="../js/uploadFile.js"></script>
+</head>
+<body>
+    <h1>Upload File (Receipt)</h1>
+    <form action="">
```
Comment (orogenic) [inline html/uploadfile.html:11]: eventually, `method="post" enctype="multipart/form-data"` will be required, I think?

**Review by orogenic — APPROVED** [review]

**Pull request merged**